crates/volta-core/src/shim.rs

```diff
@@ -89,6 +89,23 @@ pub fn create(shim_name: &str) -> Fallible<ShimResult> {
     }
 }
 
+pub fn check_reachable(shim_name: &str) -> Fallible<()> {
```

Comment:
Rather than make the public method fallible (and need to create a new error type), since this is intended to be purely informational, can we make this infallible and only show a message if we're able to successfully determine both the shim and the resolved and compare them?

Reply:
Separately, also a slight preference for moving this into the `tool/mod.rs` file, rather than holding it here in `shim.rs`. The main reason for that is we can then use the `tool` helper functions for writing the message out (e.g. `note_prefix()` and similar that are used for other notes).

Alternatively, we could have this method be something like `is_shadowed(shim_name: &str) -> ShadowStatus {}` with an enum:

```rust
pub enum ShadowStatus {
    NotShadowed,
    Shadowed {
        expected: PathBuf,
        found: PathBuf,
    }
}
```

And then only build the message inside of the `tool` module.